### DISCUSSION TITLE
Getting error while importing via Terraformer for sg

### DISCUSSION THREAD

**Discussion by BBS20** [GoogleCloudPlatform/terraformer#1679] [Q&A]
I am getting the following error when I try to import the sg using terraformer.  Could you please help me identify and resolve this issue? 
![Terraformer_sg_error](https://user-images.githubusercontent.com/88154763/227569925-f13f6ffa-fe85-4d96-8af8-39dea82ff3f3.PNG)

**Comment by Sarthak-Agarwal1410** [comment]
I'm getting same error now @BBS20 
Did you resolved it?
